packages/synapse-react-client/src/components/file/upload/MultiFileUploadProgress.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  FILE_UPLOAD_PROGRESS_COMPONENT_HEIGHT_PX,
+  FileUploadProgress,
+} from '@/components/EntityUpload/FileUploadProgress'
+import getActiveUploadCount from '@/utils/hooks/useUploadFileEntity/getActiveUploadCount'
+import { EntityUploaderState } from '@/utils/hooks/useUploadFileEntity/useUploadFileEntities'
+import { UploadItem } from '@/utils/hooks/useUploadFileEntity/useUploadFiles'
+import Box from '@mui/material/Box'
+import Paper from '@mui/material/Paper'
+import Typography from '@mui/material/Typography'
+import pluralize from 'pluralize'
+import { FixedSizeList } from 'react-window'
+
+// This padding value will be used to manipulate the appearance of a virtualized list of FileUploadProgress components
+const UPLOAD_CONTAINER_PADDING_X_PX = 24
+
+type UploadProgressProps = {
+  /**
+   * The current state of the uploader.
+   */
+  uploaderState: EntityUploaderState
+  /**
+   * The progress of each file being uploaded, including status and callbacks for user actions.
+   */
+  uploadProgress: UploadItem[]
+}
+
+export default function MultiFileUploadProgress(props: UploadProgressProps) {
```

Comment:
Identical to the React tree that displayed entity progress in `EntityUpload.tsx`, just extracted for reuse